src/main/kotlin/com/devfloor/untitled/option/application/OptionService.kt

```diff
@@ -0,0 +1,17 @@
+package com.devfloor.untitled.option.application
+
+import com.devfloor.untitled.option.domain.Option
+import com.devfloor.untitled.option.domain.OptionRepository
+import com.devfloor.untitled.option.domain.OptionType
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class OptionService(
+    private val repository: OptionRepository
+) {
+    @Transactional(readOnly = true)
+    fun showAllByOptionType(options: List<String>): List<Option> {
+        return repository.findAllByTypeIn(options.map { OptionType.from(it) })
+    }
```

Comment:
```suggestion
    fun showAllByOptionType(optionTexts: List<String>): List<Option> {
        val optionTypes = optionTexts.map { OptionType.from(it) }
        return repository.findAllByTypeIn(optionTypes)
    }
```

이런 식은 어떤가요?? 저는 개인적으로 인자로 넘어갈 때 라인이 너무 길어지지 않으면 이렇게 변수로 빼서 이름을 지어주고(여기선 optionTypes) 그걸 인자로 넘겨주면 코드가 더 잘 이해가고 좋더라구요!!